packages/use-contractkit/src/connectors/connectors-by-name.ts

```diff
@@ -24,9 +24,4 @@ export const CONNECTOR_TYPES: {
   [WalletTypes.PrivateKey]: PrivateKeyConnector,
   [WalletTypes.Unauthenticated]: UnauthenticatedConnector,
   [WalletTypes.WalletConnect]: WalletConnectConnector,
-  // TODO: only show these ones dynamically depending on the which wallets run on our blockchain (eip155:42220)
```

Comment:
Now that these have been removed, what happens if the WC registry is down or slow? It was less flexible but it felt more robust to have some amount of hard-coding here for the important wallets

Reply:
After a quick talk with @dckesler we figured that we could use the registry as an npm module as a fallback, it's implemented in @c1a0be0